guvnor-structure/guvnor-structure-backend/src/main/java/org/guvnor/structure/backend/organizationalunit/OrganizationalUnitServiceImpl.java

```diff
@@ -114,6 +114,11 @@ public OrganizationalUnit getOrganizationalUnit( final String name ) {
 
     @Override
     public Collection<OrganizationalUnit> getAllOrganizationalUnits() {
+        return new ArrayList<>( registeredOrganizationalUnits.values() );
+    }
+
+    @Override
+    public Collection<OrganizationalUnit> getOrganizationalUnits() {
```

Comment:
Could we use more descriptive name? When looking at the code, it is not obvious to me what is the difference between `getAllOrganizationalUnits()` and `getOrganizationalUnits()`. Perhaps something like `getAccessibleOrganizationUnits`?